.github/actions/linux/build-libmongocrypt/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Build libmongocrypt"
+description: "Installs libmongocrypt"
+inputs:
+  version:
+    description: "Libmongocrypt version to install (major.minor)"
+    required: true
+runs:
+  using: composite
+
+  steps:
+    - name: Add repository key
+      shell: bash
+      run: sudo sh -c 'curl -s --location https://pgp.mongodb.com/libmongocrypt.asc | gpg --dearmor >/etc/apt/trusted.gpg.d/libmongocrypt.gpg'
+
+    - name: Add repository
+      shell: bash
+      working-directory: /tmp
+      # Note: no packages for Ubuntu 24.04 noble exist, so we use those for 22.04
+      run: echo "deb https://libmongocrypt.s3.amazonaws.com/apt/ubuntu jammy/libmongocrypt/${LIBMONGOCRYPT_VERSION} universe" | sudo tee /etc/apt/sources.list.d/libmongocrypt.list
+      env:
+        LIBMONGOCRYPT_VERSION: ${{ inputs.version }}
+
+    - name: Update apt sources
+      shell: bash
+      run: sudo apt-get update
+
+    - name: Install libmongocrypt
+      shell: bash
+      run: sudo apt-get install -y libmongocrypt-dev
```

Comment:
It might be possible to cache the apt dependency: https://github.com/marketplace/actions/cache-apt-packages

Reply:
Noted for a separate PR.